parquet/src/main/java/org/apache/iceberg/parquet/ParquetSchemaUtil.java

```diff
@@ -164,4 +164,48 @@ public Boolean primitive(PrimitiveType primitive) {
     }
   }
 
+  // Parquet LIST backwards-compatibility rules.
+  // https://github.com/apache/parquet-format/blob/master/LogicalTypes.md#backward-compatibility-rules
+  public static boolean isListElementType(Type repeatedType, String parentName) {
```

Comment:
Perhaps this should be isOldListElementType or something to indicate this is for backwards compatibility in the name, we also switch the name to "isElementType" later which I think we should also rename to show this is the backwards compatibility path.

Reply:
Good point, updating to `isOldListElementType`